tests_integration_with_local_chain/conftest.py

```
@@ -57,14 +57,24 @@ def local_web3(
     print("exiting fixture local_web3")
 
 
+def eoa_accounts(local_web3: Web3, accounts: list[TestAccount]) -> list[TestAccount]:
+    # We filter out accounts that are smart accounts because our methods `send_xdai_to` fails in that case (we are using
```

Pylint (3.3.7)

[convention] Line too long (120/100)

(C0301)